src/editor/CodeHintManager.js

```diff
@@ -249,7 +249,8 @@ define(function (require, exports, module) {
         hintList         = null,
         deferredHints    = null,
         keyDownEditor    = null,
-        codeHintsEnabled = true;
+        codeHintsEnabled = true,
+        codeHintOpened   = false;
```

Comment:
Found sort of a problem with my fix, If you have hint list keyboard shortcut set to ctrl+space then when you open the list, it automatically jumps to the end of the list. One way I can think of fixing this is making codeHintOpened a boolean and when it is >= 1 then allow the ctrl+space command to work. Any better ideas or suggestions?